### PR TITLE
Improve WordPress.org support

### LINKED ISSUE #181: Create a CHANGELOG.md
It would be useful to create a `CHANGELOG.md` file that contained the canonical list of changes that developers and users of the plugin should know about. The common format is defined at http://keepachangelog.com/. This would be updated as part of the release process, looking back through the net changes since the last release. The changes for a particular release is then used in the description of the Release on GitHub.

### LINKED ISSUE #182: Consolidate readme.txt into README.md
As per https://johnblackbourn.com/readme-md-plugin-on-wordpress-org/, then `README.md` can be used for both GitHub, and act as the `readme.txt` on the WordPress.org repository - there is no need to maintain two versions.

### PR DESCRIPTION
**Pull request by GaryJones**
## Screenshots: Replace screenshots 3 and 4

The existing screenshots 3 and 4 were quite generic - they were of code snippets from a localhost environment.

`json-ld-section-screenshot.png` and `json-ld-screenshot.png` were already present in the repo and used in the readme, but these would not have been accessible to wordpress.org due to incorrect filenames.

The commit renames the `json-*.png` images to be `screenshot-3.png` and `screenshot-4.png`

See #186.

## Screenshots: Move to .wordpress-org directory
This directory will be used for all of the wordpress.org assets.

See #186.
See #187.

## Add CHANGELOG.md

- Generally follows the keepachangelog.com format, but doesn't break each release down into subcategories as there aren't enough entries to need it.
- Release dates are taken from the git tag dates, which may not match up to when changes were pushed to WordPress.org SVN.
- Changed each entry to use imperative tense.
- Added new entries for where the content was missing from the changelog section in the `README.md`.

Closes #181.

## Update `README.md` to be valid for wordpress.org

As per johnblackbourn.com/readme-md-plugin-on-wordpress-org, a `README.md` file on GitHub can be made valid for plugins on WordPress.org.

See #182.

## Remove readme.txt

Redundant now that README.md is valid for wordpress.org.

Closes #182.

## Icons: Add existing plugin icons to repo
These icons already exist on the WordPress.org SVN repo, but they didn't exist in the Git repo; this fixes that.

See #186.

## Icons: Replace with correct logotype
In 2019, the logotype was [refreshed](https://blog.parse.ly/post/9092/the-parse-ly-brand-refresh-our-10-year-glow-up/), so these icons are the updated versions. 
Here's the comparison:

Existing / Old:
Old

New:
icon-128x128

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,30 +40,17 @@ Feedback, suggestions, questions or concerns? E-mail us at [support@parsely.com]
...
-1. Checkout the branch on which you've been working
-1. Navigate back up to the root of the `vccw` directory and run `vagrant up`
-1. Ssh into the Vagrant machine with `vagrant ssh`
-1. Delete extra wordpress files in /tmp with `rm -rf /tmp/wordpress*` (see this issue for context: https://github.com/wp-cli/wp-cli/issues/1938)
-1. Run the test init script, if you haven't already: `bash bin/install-wp-tests.sh wptest root <db password>`
-1. Navigate to `/var/www/wordpress/wp-content/plugins/wp-parsely` and run the test suite with the command `phpunit`
-
-
-## Frequently Asked Questions ##
+## Frequently Asked Questions
```
Comment (hbbtstar) [inline README.md:43]: While these instructions are quite out of date, do we want to add any suggested workflow here or just don't worry about it?

**Review by hbbtstar — APPROVED** [review]
this LGTM, though I'm confused on order of operations: does #187 have to be merged first, or this one? 187 says it has to be merged first, but it looks maybe like a typo since this one has the meat of the changes.

**Comment by GaryJones** [comment]
> I'm confused on order of operations: does #187 have to be merged first, or this one?

#188 should be merged first. The Note in #187 referenced PR #187, instead of #188 (this one) - I've now fixed that note. Sorry!

File: README.md (hunk 1 above)
Comment (GaryJones) [reply README.md:43]: At this point, I was just consolidating the readme's; other changes to the content can be done afterwards in a new PR, when there's only one file to edit.

**Comment by hbbtstar** [comment]
ah gotcha! No worries, I thought so from context but never hurts to double check.

File: README.md (hunk 1 above)
Comment (hbbtstar) [reply README.md:43]: :+1: works for me

**Pull request merged**